actfw_core/task/tee.py

```diff
@@ -0,0 +1,77 @@
+import traceback
+from queue import Empty, Full, Queue
+from threading import Thread
+
+from .task import Task
+
+
+class Tee(Task):
+
+    """Tee Task."""
+
+    def __init__(self):
+        """"""
+        super(Tee, self).__init__()
+        self.running = True
+        self.in_queues = []
+        self.out_queues = []
+        self.out_queue_id = 0
+
+    def _is_running(self):
+        return self.running
+
+    def _add_in_queue(self, q):
+        self.in_queues.append(q)
+
+    def _add_out_queue(self, q):
+        self.out_queues.append(q)
+
+    def _inlet(self):
+        in_queue_id = 0
+        length = len(self.in_queues)
+        while self._is_running():
+            try:
+                i = self.in_queues[in_queue_id].get(timeout=1)
+                yield i
+                in_queue_id = (in_queue_id + 1) % length
+            except Empty:
+                pass
+            except GeneratorExit:
+                break
+            except:
+                traceback.print_exc()
+
+    def _outlet(self, o):
+        while self._is_running():
+            for out_queue in self.out_queues:
+                try:
+                    out_queue.put(o, timeout=1)
```

Comment:
This passes the same `Frame` object to some successors and lets them share one.
Doesn't it violate the meaning `Frame._update`? If someone touches it, others cannot get "fresh" frames (if slow).

Reply:
https://github.com/Idein/actfw-core/pull/16 explain why `Frame._update` is required historically. plz. fix it anothor PR.

Reply:
https://github.com/Idein/actfw-core/issues/17